### PR TITLE
Misc update #92 #95 #97

### PR DESCRIPTION
**Pull request by roxk**
connect #95
connect #92
connect #97

### CODE REVIEW

File: sdk/src/main/java/com/oursky/authgear/AuthgearCore.kt [hunk 1]
```diff
@@ -331,6 +331,7 @@ internal class AuthgearCore(
                 prompt = listOf(PromptOption.NONE),
                 loginHint = loginHint,
                 wechatRedirectURI = options?.wechatRedirectURI,
+                colorScheme = options?.colorScheme,
```
Comment (louischan-oursky) [inline sdk/src/main/java/com/oursky/authgear/AuthgearCore.kt:334]: Also support ui_locales in openUrl.

When we support ui_locales and x_color_scheme for openUrl, we need to inject them in the query. See https://github.com/authgear/authgear-sdk-flutter/pull/2/files#diff-0855c980dde3d19bfc99d65777c17622b6f977e91632f223afefb91282aeb495R275

Comment (louischan-oursky) [reply sdk/src/main/java/com/oursky/authgear/AuthgearCore.kt:334]: In case you missed this commit

Comment (roxk) [reply sdk/src/main/java/com/oursky/authgear/AuthgearCore.kt:334]: Then xamarin needs update as well. Will open issue.

Comment (roxk) [reply sdk/src/main/java/com/oursky/authgear/AuthgearCore.kt:335]: Updated, thanks. `authorizationUrl` should parse the config into queries correctly so we just need to pass ui locales here.

**Pull request merged**